src/app/body/task-details/edit-page/edit-page.component.ts

```diff
@@ -118,6 +127,29 @@ export class EditPageComponent implements OnInit {
           });
     }); 
   }
+  handleMinsHrs(){
```

Comment:
We can optimise this code by moving it into tools Service

Reply:
pushed the changes